doc/name.md

```diff
@@ -1,17 +1,17 @@
 # Faker.Name
 
 ```cs
-Faker.Name.Name()             //=> "Tyshawn Johns Sr."
+Faker.Name.Name()           //=> "Tyshawn Johns Sr."
```

Comment:
Could you revert the changes on this file? It doesn't relate to the Pokemon feature.